Emby.Dlna/Emby.Dlna.csproj

```diff
@@ -20,8 +20,7 @@
     <TargetFramework>net5.0</TargetFramework>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Leave this true for Release builds (same for all other projects)

Reply:
Yup. You are right. I didn't quite grasp what as going on with those style analyzers at first glance.